slider/README.md

```diff
@@ -3,18 +3,23 @@
  <img width=150px src="https://a.storyblok.com/f/51376/3856x824/fea44d52a9/colored-full.png" alt="Storyblok" style="display:inline-block;margin:20px;"></a>
 </p>
 
-# ![icon.svg](docs/icon.svg) storyblok-palette
+<h1>
+<img src="assets/icon.svg" height="26px">
+Slider Field Plugin
+</h1>
 
-A custom field-type for Storyblok that allow users to select a color from a palette. 
```

Comment:
Throughout the file there are three name versions used for field-plugin --> field, field type, and plugin. Do we maybe want to start to use field-plugin here for consistency?

Reply:
I'll create a second PR soon